ydb/core/tx/datashard/prefix_kmeans.cpp

```diff
@@ -298,26 +303,26 @@ class TPrefixKMeansScanBase: public TActor<TPrefixKMeansScanBase>, public NTable
         Driver->Touch(EScan::Final);
     }
 
-    EScan FeedUpload()
+    ui64 GetProbability()
+    {
+        return Rng.GenRand64();
+    }
+
+    bool ShouldWaitUpload()
```

Comment:
All function can be written like (because how you implemented `TryUpload`)
```cpp
return  TryUpload(LevelBuf, LevelTable, LevelTypes, true)
     || TryUpload(PostingBuf, PostingTable, PostingTypes, true)
     || TryUpload(PrefixBuf, PrefixTable, PrefixTypes, true);
```

Reply:
I think no, TryUpload return true when something is uploading

But here we should return true when something is uploading and some other buffer reaches its limit